src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/AdoNet/AdoNetExtensions.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AdoNet.SqlClient
+{
+    internal static class AdoNetExtensions
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static void DisposeWithException(this Scope scope, Exception exception)
```

Comment:
This seems like a nice helper to use in most integrations. Is it worth moving it out of the `AdoNet` directory?

Reply:
You're right. Done in 70318d8 :)